src/Pelagos/Bundle/AppBundle/Security/LoginFormAuthenticator.php

```diff
@@ -63,24 +64,34 @@ class LoginFormAuthenticator extends AbstractFormLoginAuthenticator
      */
     protected $logger;
 
+    /**
+     * String describing max PW age.
+     *
+     * @var string maximumPasswordAge
+     */
+    protected $maximumPasswordAge;
+
     /**
      * Class constructor for Dependency Injection.
      *
-     * @param FormFactoryInterface   $formFactory   A Form Factory.
-     * @param EntityManagerInterface $entityManager An Entity Manager.
-     * @param RouterInterface        $router        A Router.
-     * @param LoggerInterface        $logger        A Monolog logger.
+     * @param FormFactoryInterface   $formFactory        A Form Factory.
+     * @param EntityManagerInterface $entityManager      An Entity Manager.
+     * @param RouterInterface        $router             A Router.
+     * @param LoggerInterface        $logger             A Monolog logger.
+     * @param string|null            $maximumPasswordAge The max age for password, 0 or null means never expires.
      */
     public function __construct(
         FormFactoryInterface $formFactory,
         EntityManagerInterface $entityManager,
         RouterInterface $router,
-        LoggerInterface $logger
+        LoggerInterface $logger,
+        $maximumPasswordAge
```

Comment:
You can use `? string` here i think.

Reply:
We usually do something like this right?
`string $maximumPasswordAge=null`